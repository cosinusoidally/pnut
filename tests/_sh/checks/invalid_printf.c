// expect_comp_failure
void main() {
  printf("%d %d %d", 1, 2); // Too few arguments
}
