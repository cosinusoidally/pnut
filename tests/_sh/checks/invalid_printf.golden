tests/_sh/checks/invalid_printf.c:5:0  Not enough parameters for printf
